src/test/java/uk/gov/hmcts/reform/ccd/documentam/controller/endpoints/CaseDocumentAmControllerTest.java

```diff
@@ -89,22 +89,15 @@ public void setUp() {
 
     @Test
     public void shouldGetValidMetaDataResponse() {
-        Optional<DocumentPermissions> documentPermissions = Optional.ofNullable(getDocumentPermissions(
-            MATCHED_DOCUMENT_ID,
-            Arrays.asList(
-                Permission.CREATE,
-                Permission.READ
-            )
-        ));
-        doReturn(setDocumentMetaData()).when(documentManagementService).getDocumentMetadata(getUuid());
+        doReturn(Optional.of(setDocumentMetaData())).when(documentManagementService).getDocumentMetadata(getUuid());
```

Comment:
This is called in nearly every test method - would it be better off in `setUp()` ?